frontend/www/js/omegaup/components/problem/creator/Creator.vue

```diff
@@ -1,9 +1,19 @@
 <template>
-  <div>Creator Path Test</div>
+  <b-container fluid="lg">
+    <Header />
+    <Tabs />
+  </b-container>
 </template>
 
 <script lang="ts">
 import { Component, Vue } from 'vue-property-decorator';
-@Component({})
+import Header from './Header.vue';
+import Tabs from './Tabs.vue';
+@Component({
+  components: {
+    Header,
+    Tabs,
```

Comment:
```suggestion
    'creator-header': Header,
    'creator-tabs': Tabs,
```

igual acá. y además el estándar require el no-uso de mayúsculas.